client/performance-profiler/pages/loading-screen/index.tsx

```diff
@@ -8,50 +9,238 @@ interface LoadingScreenProps {
+		position: relative;
+
+		.gridicons-checkmark {
+			display: none;
+		}
+
+		&:before {
+			content: '';
+			display: inline-block;
+			width: 30px;
+			height: 30px;
+			border-radius: 50%;
+			margin-right: 10px;
+			border: 1px solid transparent;
+		}
+
+		@keyframes rotate {
+			from {
+				transform: rotate( 0deg );
+			}
+			to {
+				transform: rotate( 360deg );
+			}
+		}
+
+		@-webkit-keyframes rotate {
+			from {
+				-webkit-transform: rotate( 0deg );
+			}
+			to {
+				-webkit-transform: rotate( 360deg );
+			}
+		}
+
+		&.complete {
+			.gridicons-checkmark {
+				display: block;
+				position: absolute;
+				top: 8px;
+				left: 7px;
+				fill: #fff;
+			}
+			:before {
+				background-color: var( --studio-green-30 );
+				color: #fff;
+			}
+		}
+
+		&.running:before {
+			border: 1px solid var( --studio-gray-5 );
+		}
+
+		&.running:after {
+			content: '';
+			width: 30px;
+			height: 30px;
+			position: absolute;
+			left: 0;
+			top: 0;
+			border: solid 1.5px rgba( 56, 88, 233, 1 );
+			border-radius: 50%;
+			border-right-color: transparent;
+			border-bottom-color: transparent;
+			-webkit-transition: all 0.5s ease-in;
+			-webkit-animation-name: rotate;
+			-webkit-animation-duration: 1s;
+			-webkit-animation-iteration-count: infinite;
+			-webkit-animation-timing-function: linear;
+
+			transition: all 0.5s ease-in;
+			animation-name: rotate;
+			animation-duration: 1s;
+			animation-iteration-count: infinite;
+			animation-timing-function: linear;
+		}
+
+		&.incomplete {
+			color: var( --studio-gray-20 );
+			::before {
+				border: 1px dashed var( --studio-gray-5 );
+			}
+		}
 	}
 `;
 
-const StyledLoadingScreen = styled.div`
-	padding: 100px 40px;
+const FootNote = styled.div`
+	max-width: 460px;
+	background-color: #e7f0fa;
+	padding: 25px;
+	margin-top: 65px;
 
-	h2 {
-		text-align: center;
+	h4 {
+		font-size: 14px;
+		font-weight: 500;
+		line-height: 20px;
+		margin-bottom: 10px;
+	}
+
+	p {
+		font-size: 14px;
+		margin-bottom: 0;
+	}
+
+	.learn-more-link {
+		margin-top: 20px;
 	}
 `;
 
 export const LoadingScreen = ( { isSavedReport }: LoadingScreenProps ) => {
 	const translate = useTranslate();
+	const [ step, setStep ] = useState( 0 );
 
-	const progressHeadings = isSavedReport
+	const steps = isSavedReport
 		? [ translate( 'Getting your report…' ) ]
 		: [
-				translate( "Crunching your site's numbers…" ),
-				translate( 'Analyzing speed metrics…' ),
-				translate( 'Comparing with top sites…' ),
-				translate( 'Finalizing your report…' ),
+				translate( 'Loading your site' ),
+				translate( 'Measuring Core Web Vitals' ),
+				translate( 'Taking screenshots' ),
+				translate( 'Fetching historic data' ),
+				translate( 'Identifying performance improvements' ),
+				translate( 'Finalizing your results' ),
 		  ];
 
-	const [ progress, setProgress ] = useState( 0 );
-	const [ tick, setTick ] = useState( 0 );
+	const tips = [
+		{
+			heading: translate( 'Performance Mattters' ),
```

Comment:
Assuming this is a typo (and not a play on Matt's nam ;-) ). 

`Mattters` --> `Matters`

Reply:
Thanks for the callout! Fixed in https://github.com/Automattic/wp-calypso/pull/93660